jekyll/_cci2/enable-checks.md

```diff
@@ -9,77 +9,84 @@ version:
 - Cloud
 ---
 
-This document describes how to enable the GitHub Checks CircleCI Setting and authorize the CircleCI Checks app to report workflow status to the GitHub app. The GitHub checks integration feature is not currently available on CircleCI Server.
+This document describes how to enable the GitHub Checks feature and authorize CircleCI to report workflow status to the GitHub app. **The GitHub checks integration feature is not currently available on CircleCI Server**.
+
+## GitHub Check and GitHub status updates
+
+GitHub Checks should not be confused with GitHub status updates:
+
+* GitHub Checks are administered from the GitHub UI, and are reported in the GitHub UI per _workflow_.
+* GitHub status updates are the default way status updates from your builds are reported in the GitHub UI, and they are reported per _job_. 
+
+If both these features are enabled, the Checks tab in your GitHub PRs will show both job and workflow statuses.
```

Comment:
```suggestion
If both these features are enabled, in a GitHub PR view the Checks tab will show workflow status and the Checks section in the PR conversation view will show job status.
```

Reply:
just a little further clarification - it's tricky because there are effectively two github "things" called Checks...